src/main/webapp/js/finscholar/scholarshiplistview/scholarshiplistview.js

```diff
@@ -15,21 +15,30 @@
 /** @fileoverview The mini controller for scholarship list view. */
 
 goog.module('finscholar.scholarshiplistview');
-const {scholarshiplist} = goog.require('finscholar.scholarshiplistview.templates');
+
+const {CommonListView} = goog.require('finscholar.commonlistview');
+const {ScholarshipListDataHandler} = goog.require('datahandlers.scholarshiplistdatahandler');
+
+const SCHOLARSHIP_LIST_INDEX = '1';
 
 /** The mini controller for scholarship list view. */
-class ScholarshipListView {
+class ScholarshipListView extends CommonListView {
   
   constructor() {
-    // TODO: In MVP, we'll add member variables such as pages, the array of scholarship etc.
+    super(new ScholarshipListDataHandler(), SCHOLARSHIP_LIST_INDEX);
```

Comment:
Why not make the dataHandler part of this class? Then you can override methods from the parent.